File: Calling/app/src/main/java/com/example/calling/MainActivity.java
package com.example.calling;

import androidx.appcompat.app.AppCompatActivity;
import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.ImageView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

   EditText te;
   ImageView iv;
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        te=(EditText)findViewById(R.id.editTextPhone);
        iv=(ImageView)findViewById(R.id.imageView);


        iv.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                String number = te.getText().toString();
                if(number.trim().length()==10){
                    String dial = "tel:" + number;
                    startActivity(new Intent(Intent.ACTION_CALL, Uri.parse(dial)));
                }
                else{
                    Toast.makeText(getApplicationContext(),"Enter Valid Number",Toast.LENGTH_LONG).show();
                }
            }
        });
    }
}

File: kotlinAssignment/app/src/main/java/com/example/kotlinassignment/MainActivity.kt
package com.example.kotlinassignment

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val textView = findViewById<TextView>(R.id.textView2)
        val button = findViewById<Button>(R.id.button)

        button.setOnClickListener(){
            intent = Intent(this,MainActivity2::class.java)
            startActivity(intent)
            Toast.makeText(this,"Going to 2nd Activity", Toast.LENGTH_LONG).show()
        }
    }
}